src/Tests/Singularity.Test/Extensions/ModuleExtensionsTests.cs

```diff
@@ -13,5 +13,16 @@ public void ToBindings_SingleModule()
 			var instance = container.GetInstance<ITestService10>();
 			Assert.IsType<TestService10>(instance);
 		}
+
+        [Fact]
+        public void RegisterModule_Generic()
```

Comment:
For readability, please use ARRANGE ACT ASSERT comments to clearly separate different phases in the tests.

Reply:
No problem. I have just submitted a change to address this.